controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -123,17 +123,26 @@ public ResponseEntity<Object> fetchPdf(String claimSubmissionId)
   public ResponseEntity<FullHealthDataAssessmentResponse> postFullHealthAssessment(
       HealthDataAssessmentRequest claim)
       throws RequestValidationException, ClaimProcessingException {
-    log.info("Getting health assessment for: {}", claim.getVeteranIcn());
+    log.info(
+        "Getting full health assessment for claim {} and veteran icn {}",
+        claim.getClaimSubmissionId(),
+        claim.getVeteranIcn());
     try {
       Claim model = postClaimRequestMapper.toModel(claim);
       String responseAsString = camelEntrance.submitClaimFull(model);
-      log.info("Obtained full health assessment: {}", responseAsString);
+
       FullHealthDataAssessmentResponse response =
           objectMapper.readValue(responseAsString, FullHealthDataAssessmentResponse.class);
+      if (response.getEvidence() == null) {
+        throw new ClaimProcessingException(
+            claim.getClaimSubmissionId(), HttpStatus.NOT_FOUND, "No evidence found.");
+      }
       log.info("Returning health assessment for: {}", claim.getVeteranIcn());
       response.setVeteranIcn(claim.getVeteranIcn());
       response.setDiagnosticCode(claim.getDiagnosticCode());
       return new ResponseEntity<>(response, HttpStatus.CREATED);
+    } catch (ClaimProcessingException cpe) {
```

Comment:
The rethrow ClaimProcessingException should not be needed here, should it?

Reply:
When I remove it I get `500 INTERNAL_SERVER_ERROR` for the missing evidence test

Reply:
Instead of `NOT_FOUND`